### ISSUE TITLE
new feature -- including syllable number into the schema

### ISSUE THREAD

**Issue by mattdzugan** [skirmer/radlibs#1]
what would you think about allowing for something like:

```
> radlibs::makeRadlibs("who verbs_1 in a noun_3 under the noun_1?")
[1] "Who lives in a pineapple under the sea?"
```

instead of 
```
> radlibs::makeRadlibs("who verbs in a noun under the noun?")
[1] "Who freights in a tear under the courtship?"
```

**Comment by skirmer** [comment]
Ok, so I'm interested, but there is a big issue with identifying the number of syllables in the pool of words that the function calls from. I don't know of a dataset that contains this, or a function that could extract the number of syllables in the words. Do you? If you can give me a way to find out the number of syllables in words in the humor dataset I'm open to it.

**Comment by mattdzugan** [comment]
there's gotta be ways to do it - i like a challenge!
just wanted to see if you were open to it.

we'll have to hash out the "syntax" as well:
`noun_1` vs `noun---1` vs `noun1` vs `noun^1` or whatever

**Comment by skirmer** [comment]
Resolved with version 0.2.0! #4
